app/views/registered_applications/destroy.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>RegisteredApplications#destroy</h1>
```

Comment:
Review the templates within views/registered_applications and remove any that aren't needed.